### PR TITLE
Create postcss.config.js

### LINKED ISSUE #171: Can not run dev environment
Potentially an issue isolated to Apple M-series chips.

Error on M1 (Pro).

Screen Shot 2022-11-04 at 09 42 11

### PR DESCRIPTION
**Pull request by ShortCipher5**
This would fix #171.

### CODE REVIEW

**Comment by ShortCipher5** [comment]
Added a configuration to disable ESLint rules for *.config.js

File: postcss.config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
```
Comment (doodlewind) [inline postcss.config.js:1]: I think we're not using it for now, so there may be another root cause for #171

**Comment by ShortCipher5** [comment]
As it seems that tailwind is not used I removed the entry from the config.

A blank postcss.config.js does not seem to work, but an empty module export does:
`module.exports = {}`

**Comment by ShortCipher5** [comment]
> As it seems that tailwind is not used I removed the entry from the config.
> 
> A blank postcss.config.js does not seem to work, but an empty module export does: `module.exports = {}`

Actually, as long as the file is not empty it can work - so even an empty comment `//` is ok.

**Pull request closed without merging**